parsers/ccmpredparser.py

```diff
@@ -0,0 +1,28 @@
+from utils.exceptions import InvalidFormat
+from operator import itemgetter
+
+def CCMpredParser(input):
+    contents = input.split('\n')
+
+    output = []
+
+    data = []
+    for line in contents:
+        line = line.lstrip().split('\t')
+        if not line or line[0].isalpha():
+            continue
+        else:
+            data.append(line)
+
+    for res_1, score_array in enumerate(data):
+        for res_2, score in enumerate(score_array):
+            if abs((res_1+1) - int(res_2+1)) >= 5 and score!='':
```

Comment:
Also, CCMPred files are a matrix of contacts, so even contacts with P=0 are listed in the file... Let's add only those contacts with a P>0. That would be `if abs((res_1) - int(res_2)) >= 5 and score!='' and float(score)>0:`

Reply:
When you implement this you will probably need to adjust the expected output of the tests